### PR TITLE
Refactoring zustand

### PR DESCRIPTION
**Pull request by DerElchausdemAll**

### CODE REVIEW

**Comment by konrascher3** [comment]
Bis auf den Test schaut alles gut aus, ich find dein Zustand ganz geil mit der fetch und updateData.

File: src/organisms/card/Card.jsx [hunk 1]
```diff
@@ -88,4 +83,10 @@ const NotesCard = ({ note }) => {
...
 };
 
-export default NotesCard;
+export default ItemsCard;
+
+// fetch(`http://localhost:8000/items/${item.id}`, {
+// 	method: "PUT",
+// 	headers: { "Content-type": "application/json" },
+// 	body: JSON.stringify({ ...item, checked: nextChecked }),
+// });
```
Comment (doemser) [inline src/organisms/card/Card.jsx:92]: consider putting code thats not running into scratch files

**Review by doemser — APPROVED** [review]
whoops, some checks did not pass. :o

**Comment by doemser** [comment]
in your package.json:
`"test": "npm run test:unit && npm run test:e2e"`
should be:
`"test": "npm run test:unit"`

you can reactivate that later, if you're planning on using e2e tests for your components.

**Pull request merged**